php/libraries/User.class.inc

```diff
@@ -362,6 +362,45 @@ class User extends UserPermissions
         return $user_study_sites;
     }
 
+    /**
+     * Returns a list with the user's sites
+     *
+     * @param boolean $study_site if true only return study sites
+     * @param boolean $DCC        Whether the DCC should be included or not
+     *
+     * @return array an associative array("center ID" => "site name")
+     */
+    function getSiteList(
```

Comment:
As i suggested in the PR below I think the function name should be getSiteNameList() since this doesnt return sites but names of sites

https://github.com/aces/Loris/pull/5892/files#r406274087